src/main/java/racinggame/domain/racing/RacingCars.java

```diff
@@ -1,23 +1,70 @@
 package racinggame.domain.racing;
 
 import racinggame.domain.car.Car;
+import racinggame.util.NumberUtil;
 
+import java.util.ArrayList;
 import java.util.List;
 
 /**
  * 1. 레이싱을 진행하는 car들의 collection
  * 1. 모든 자동차를 이동
  */
 public class RacingCars {
+    private static final int RANDOM_BOUND = 10;
+
     private final List<Car> cars;
 
     public RacingCars(List<Car> cars) {
-        this.cars = cars;
+        List<Car> result = new ArrayList<>();
+
+        for (Car car : cars) {
+            result.add(new Car(car));
+        }
+
+        this.cars = result;
```

Comment:
`Car`가 VALUE OBJECT라면 아래의 코드가 가능합니다. 왜 그럴까요?

```suggestion
        this.cars = new ArrayList<>(cars);
```